src/wallet/coinselection.h

```diff
@@ -28,12 +28,20 @@ class CInputCoin {
         effective_value = txout.nValue;
     }
 
+    CInputCoin(const CTransactionRef& tx, unsigned int i, int input_bytes) : CInputCoin(tx, i)
+    {
+        m_input_bytes = input_bytes;
+    }
+
     COutPoint outpoint;
     CTxOut txout;
     CAmount effective_value;
     CAmount fee = 0;
     CAmount long_term_fee = 0;
 
+    /** Pre-computed estimated size of this output as a fully-signed input in a transaction. Can be -1 if it could not be calculated */
+    int m_input_bytes{-1};
```

Comment:
what makes this different from
```
int m_input_bytes = -1;
```
(my guess is that it is the same, in which case I think that syntax is preferable because it is already used in so many places)

Reply:
It is the same. We already use this syntax in a variety of places. See e.g. 

https://github.com/bitcoin/bitcoin/blob/7ebd8c6385253bfa30175169a22f93474d8ddfaf/src/wallet/wallet.h#L97-L101

https://github.com/bitcoin/bitcoin/blob/7ebd8c6385253bfa30175169a22f93474d8ddfaf/src/wallet/wallet.h#L674-L675

Reply:
okay! well then my only comment is that mixing the syntaxes in the same class looks inconsistent, but I don't care strongly

Reply:
Oh, hum. I actually thought the {} syntax was preferred and didn't fix the other stuff to avoid diff bloat.

Reply:
Oh, actually, the other two lines are removed at the end as they are no longer needed.